app/controllers/citizens/additional_accounts_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Citizens
+  class AdditionalAccountsController < BaseController
+    def index; end
+
+    def create
+      case params[:additional_account]
+      when 'yes'
+        redirect_to new_citizens_additional_account_path
+      when 'no'
+        # TODO: - set redirect path when known
+        render plain: 'Landing page: Next step in Citizen journey'
+      else
+        flash[:error] = 'You must select either Yes or No'
+        render :index
+      end
+    end
+
+    def new; end
+
+    def update
+      case params[:additional_account]
+      when 'yes'
+        # TODO: - set redirect path when known
+        render plain: 'Landing page: Return to True Layer steps'
+      when 'no'
```

Comment:
**[not a blocker]** I personally find it a bit confusing to use the same form for two different types of resource (they're both have a Yes/No radios, that's true), but then performing this something like:

```ruby
if params[:additional_account] == 'no'
  current_legal_aid_application&.update(has_offline_accounts: true)
end
```

Wouldn't be simpler and clearer to submit this data as `params[:has_offline_accounts]` ? which expresses exactly what we're asking?